appyx-interactions/common/src/commonMain/kotlin/com/bumble/appyx/transitionmodel/BaseInterpolator.kt

```diff
@@ -96,6 +95,40 @@ abstract class BaseInterpolator<NavTarget : Any, ModelState, Props>(
         }
     }
 
+    private fun onAnimationStarted(
+        elementProps: Props,
+        targetProps: MatchedProps<NavTarget, Props>
+    ) {
+        val isStarted = animations.getOrDefault(targetProps.element.id, false)
+        if (elementProps.isEqualTo(targetProps.props).not() && !isStarted) {
+            Logger.log(
+                this@BaseInterpolator.javaClass.simpleName,
+                "animation started for element ${targetProps.element.id.substring(0..4)}"
```

Comment:
```suggestion
                "Animation started for element ${targetProps.element.id.substring(0..4)}"
```

For more uniform log messages